src/SFML/Audio/SoundFileReaderWav.cpp

```diff
@@ -88,6 +89,16 @@ namespace
     }
 
     const sf::Uint64 mainChunkSize = 12;
+
+    enum WaveFormat
+    {
+        WaveFormatPcm = 1,
+        WaveFormatExtensible = 65534,
```

Comment:
Extra comma is C++11 feature.

Reply:
Pretty sure that's C++98/C99. I definitely used it before C++11.

Reply:
from clang:

```
warning: commas at the end of enumerator lists are a C++11 extension [-Wc++11-extensions]
        WaveFormatExtensible = 65534,
```

;-)